### DISCUSSION TITLE
Scale Runelite UI elements independently from windows setting

### DISCUSSION THREAD

**Discussion by Derek123k** [runelite/runelite#17991] [Ideas]
Add a setting under the runelite configuration to scale the application UI. This should have the same function as changing the 'Scale and Layout' in windows, with the benefit of not affecting the scaling on other apps. This is not the same as the 'Stretched Mode' plugin.

On my ultrawide monitor, I have a very hard time reading the Runelite elements on standard scaling (Notes, quest helper, loot panel, etc), so I changed my windows scaling to 150%. This is not ideal because this setting affects all apps on the monitor. Some apps such as chrome and discord allow you to scale UI elements, but others such as the Jagex launcher do not.

Changing the windows scaling also affects the mouse sensitivity.

This https://github.com/runelite/runelite/discussions/14945 thread suggests changing the runelite .exe launch parameters but I don't believe this can be done through the jagex launcher. This also wouldn't be ideal if you find yourself needing to change between scaling settings.

**Comment by YvesW** [comment]
If you want to scale the sidebar, set scale to whatever value you want: https://github.com/runelite/runelite/wiki/RuneLite-Launcher-Configuration

**Reply by Derek123k** [reply]
This is exactly the setting I am looking for. I wish I knew about it earlier- I'm sure I'm not the only one who's spent too long trying to find this. I imagine the reason it's not a setting within the client is because this can only be changed on launch? 

My suggestion to make this setting more obvious to the user would be to add it inside the client configuration. If the setting can only be changed on launch, notify the user the app needs to be restarted to apply settings.

**Reply by Adam-** [reply]
It has to be set *before launch* which makes it complex as the client can't affect the way it is launched. So it is a launcher option.

**Reply by Derek123k** [reply]
I'm curious why the Runelite client couldn't affect the scale launch setting? Is this a software best practices to keep these setting external or is there something that the Launcher Configurator is capable of doing that the Runelite client cannot?
